lib/datadog/core/crashtracking/component.rb

```diff
@@ -83,6 +92,19 @@ def stop
 
         private
 
+        def register_runtime_stack_callback
+          return unless runtime_stacks_supported?
+
+          # Even if runtime stacks callback registration fails, we shouldn't block crashtracker
+          success = Crashtracking::RuntimeStacks._native_register_runtime_stack_callback
+
+          return if success
+
+          logger.debug('Failed to register runtime stack callback: registration returned false')
+        rescue => e
+          logger.debug("Failed to register runtime stack callback: #{e.message}")
+        end
```

Comment:
Minor: I think the `RuntimeStacks` class doesn't particularly add a lot at this point -- maybe just hook the native methods onto `Crashtracking::Component`?